app/seeders/basic_data/base_role_seeder.rb

```diff
@@ -74,7 +72,9 @@ def permissions(value)
       when Array
         value
       when :all_assignable_permissions
-        Roles::CreateContract.new(Role.new, nil).assignable_permissions.map(&:name)
+        Roles::CreateContract.new(self.class.model_class.new, nil)
+                             .assignable_permissions
+                             .map(&:name)
```

Comment:
I don't know if this is a bug or intended but this would always assign all `assignable_permissions` for Project Roles even if a `GlobalRole` is being seeded. @cbliard

Reply:
That's not what I am observing. I tried it and it returned the expected global permissions.

We can have a look together. Ping me when you're available.